packages/contracts/contracts/voting/allowlist/AddressListVotingManager.sol

```diff
@@ -0,0 +1,116 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.10;
+
+import "../../core/IDAO.sol";
+import "../../plugin/PluginManager.sol";
+import "./AllowlistVoting.sol";
+
+contract AddressListVotingManager is PluginManager {
```

Comment:
`AllowlistVoting` vs. `AddressListVotingManager`

Reply:
true, the plugin renaming will be handled in it's own task